src/store/cockpitApi.ts

```diff
@@ -8,10 +8,12 @@ import path from 'path';
 // We also needed to create an alias in vitest to make this work.
 import cockpit from 'cockpit';
 import { fsinfo } from 'cockpit/fsinfo';
-import toml from 'toml';
+import { read_os_release } from 'os-release';
```

Comment:
Oh neat, where did this come from? I was trying to do something similar for the dropdown in the wizard

Reply:
Nevermind, just realised it's part of cockpit

Reply:
indeed, it's part of cockpit. Though i think i'll make distribution not required in the cloudapi, that way we can drop it here at least.